app/javascript/components/project-submissions/containers/user-project-submissions-container.js

```diff
@@ -23,9 +23,16 @@ const ProjectSubmissions = (props) => {
       }
     );
     if (response.status === 200) {
-      setSubmissions(prevSubmissions =>
-        Object.assign([], prevSubmissions, {[0]: response.data})
-      );
+      const updatedSubmission = response.data;
+
+      setSubmissions(prevSubmissions => {
+        return prevSubmissions.map(previousSubmission => {
+          if (previousSubmission.id === updatedSubmission.id) return updatedSubmission;          
+
+          return previousSubmission;
+        });
+        //Object.assign([], prevSubmissions, {[0]: response.data})
```

Comment:
Can this be removed now? :)

Reply:
Yeah haha, just wanted to keep a reference of what was while folks too a look.

Reply:
Removed